samples/performance/multithreading_render_passes/multithreading_render_passes_tutorial.md

```diff
@@ -0,0 +1,103 @@
+
+A better approach is to use secondary level command buffers, which helps to reduce the overhead and get even better performance. First, both of the passes are recorded into two separate secondary command buffers using two threads. Then, we can just reference them in the primary command buffer via ``vkCmdExecuteCommands``.
+
+When using both of these methods for multi-threading, general recommendations should still be taken into account (see [Multi-threaded-recording](https://github.com/KhronosGroup/Vulkan-Samples/blob/master/samples/performance/command_buffer_usage/command_buffer_usage_tutorial.md#Multi-threaded-recording)).
+
+This sample shows the difference between recording both render passes into a single command buffer in one thread and using the methods described above.
+
+Below are screenshots of the sample running on a phone with a Mali G72 GPU:
+
+![Single Thread](images/no_multi_threading.png)
+
+Using two threads gives a 10ms frame time improvement and CPU cycles show an increase in CPU utilization:
+
+![Primary Command Buffers](images/primary_command_buffers.png)
+
+With secondary command buffers you can see a further drop in frame time.
+
+![Secondary Command Buffers](images/secondary_command_buffers.png)
+
+## Profiling
+
+Android Profiler can help to see how threads are utilized. Flame Chart shows how much time was spent for each function execution during a particular timeframe. In this particular example total contribution of command buffers recording in the main thread is 9.94 seconds within a 10 seconds capture with multi-threading disabled. 
+
+![Profiler Capture No Multithreading](images/android_studio_capture_no_multithreading.png)
+_Multithreading is disabled_
+
+With multi-threading enabled it remains almost the same (9.92s), but in the second thread 5.7s was spent for shadow pass recording. 
+
+![Profiler Capture Secondary Command Buffers](images/android_studio_capture_secondary_buffers.png)
+_Multithreading is enabled and the timeline shows how both threads are used_
+
+That means, two threads perform the same amount of work in 10s as one thread in more than 15.7 seconds and we should expect approximately 1.57 times better performance. 
+
+And indeed, in debug build, which was used for profiling, frame time is decreased from 531.1ms to 337.7ms using multi-threading (1.57 times decrease).
+
+## Further reading
+
+[Command buffer usage and multi-threaded recording](../command_buffer_usage/command_buffer_usage_tutorial.md)
+
+## Best practice summary
+
+**Do**
+
+* Use multi-threading for command buffer recording if possible.
+* Spread the workload between threads as equally, as possible, to utilize them in the most efficient way.
+
+**Avoid**
+
+* Submitting multiple primary command buffers to the queue at once, when secondary command buffers can be used.
```

Comment:
Arm specific again

Reply:
I have removed it. However, it leaves us with no "Avoid" section. I don't know how bad it is, probably I should think of something else for a substitution.